src/chocolatey/infrastructure.app/services/ChocolateyPackageInformationService.cs

```diff
@@ -59,28 +61,56 @@ public ChocolateyPackageInformation get_package_information(IPackage package)
                 return packageInformation;
             }
 
-            FaultTolerance.try_catch_with_logging_exception(
-                () =>
+            var deserializationErrorMessage = @"
+A corrupt .registry file exists at {0}.
+ Open this file in a text editor, and remove/escape any characters that
+ are regarded as illegal within XML contents.  These are typically the
+ characters <, >, "", ', and &.  Once these have been corrected, rename

```

Comment:
Just a small note, the file is XML, so it's going to have a lot of `<` and `>` items - this isn't the best advice to tell folks to remove them.

Reply:
I know what you meant, but other folks won't

Reply:
My thought process here was that I said `XML Contents` meaning attributes, elements, etc.  Perhaps that was too subtle?